content/en/docs/concepts/security/rbac-good-practices.md

```diff
@@ -0,0 +1,101 @@
+
+Ideally, pods shouldn't be assigned service accounts granted powerful permissions (listed [here](#Kubernetes-RBAC---Privilege-Escalation-Risks)). In cases where a workload requires powerful permissions, consider the following best practices:
+
+ - Limit the number of nodes running powerful pods. Avoid running powerful Daemonsets to ensure nodes are least-privileged and to limit the blast radius of container escapes.
+ - Avoid running powerful pods alongside untrusted or publicly-exposed ones. Consider using [Taints and Toleration](/docs/concepts/scheduling-eviction/taint-and-toleration/), [NodeAffinity](/docs/concepts/scheduling-eviction/assign-pod-node/#node-affinity), or [PodAntiAffinity](/docs/concepts/scheduling-eviction/assign-pod-node/#inter-pod-affinity-and-anti-affinity) to ensure powerful pods don't run alongside untrusted ones.
+
+### Hardening
+
+Kubernetes defaults to providing access which may not be required in every cluster. Reviewing the RBAC rights provided by default can provide opportunities for security hardening. Whilst, in general, changes should not be made to rights provided to `system:` accounts some options to harden cluster rights exist :- 
+
+- Review bindings for the `system:unauthenticated` group and remove where possible, as this gives access to anyone who can contact the API server at a network level.
+- Set service accounts not to mount tokens by default using the `automountServiceAccountToken: false` setting (additional information [here]/docs/tasks/configure-pod-container/configure-service-account/#use-the-default-service-account-to-access-the-api-server)). As setting this value in a pod will overwrite the service account setting, workloads which require service account tokens can still mount them.
+
+### Periodic Review
+
+It is vital to periodically review Kubernetes RBAC assignments for redundant entries, to avoid possible privilege escalation issues. If an attacker is able to create a user account with the same name as a deleted one, they will automatically inherit all the rights that that previous account had (where the right was assigned to the username specifically).
+
+## Kubernetes RBAC - Privilege Escalation Risks 
+
+Within Kubernetes RBAC there are a number of privileges which, if granted, can allow for a user or service account to escalate their rights in the cluster or affect systems outside the cluster.
+
+This document is intended to provide visibility of the areas where cluster administrators should take care, to ensure that they do not inadvertantly allow for more access to clusters than intended.
+
+### Secrets Objects - List
+
+Whilst it is generally clear that allowing **GET** access to secrets will allow a user to read their contents, which then allows for them to be re-used, it's important to note that **LIST** access effectively also allows for this, as when a list request is returned (e.g. via `kubectl get secrets -A -o yaml`) it will include the contents of the secret.
```

Comment:
```suggestion
It is generally clear that allowing **Get** access on Secrets will allow a user to read their contents.
It is also important to note that the **List** access also effectively allows for users to reveal the Secret contents.
When a List response is returned (for example, via `kubectl get secrets -A -o yaml`), the response
includes the contents of all Secrets.
```

Reply:
If it's an HTTP GET, write `GET` in code style; if it's **get** at the Kubernetes API level, write it lower case and bold. A similar story with **list**.

Aside: the response to a **list** request is usually a _collection_. If you list Secrets, you get back a collection of Secrets.